README.md

```diff
@@ -42,6 +42,27 @@ very good at that!
 
 Signed,
 
-[GitHub users](https://docs.google.com/spreadsheets/d/1oGsg02jS-PnlIMJ3OlWIOEmhtG-udTwuDz_vsQPBHKs/edit?usp=sharing)
+* Adam Bradley (@adamdbradley), maintainer of Ionic
+* Addy Osmani (@addyosmani), maintainer of TodoMVC
+* Andreas Tolfsen (@andreastt), maintainer of Selenium
+* Ariya Hidayat (@ariya), maintainer of PhantomJS
+* Cătălin Mariș (@alrra), maintainer of HTML5 Boilerplate
+* Christopher Chedeau (@vjeux), maintainer of React Native
+* Dave Methvin (@dmethvin), maintainer of jQuery
+* Domenic Denicola (@domenic), maintainer of WHATWG Standards
+* Forbes Lindesay (@ForbesLindesay), maintainer of Pug (formally known as Jade)
+* James Kyle (@thejameskyle), maintainer of Babel
+* Henry Zhu (@hzoo), maintainer of JSCS
```

Comment:
Maybe you mean babel-eslint which isn't the same; also it made sense just to put 1 project.